modules/microshift-auto-recovery-example-rpm-systems.adoc

```diff
@@ -28,8 +25,11 @@ $ sudo mkdir -p /usr/lib/systemd/system/microshift.service.d
 $ sudo tee /usr/lib/systemd/system/microshift.service.d/10-auto-recovery.conf > /dev/null <<'EOF'
 [Unit]
 OnFailure=microshift-auto-recovery.service
+StartLimitIntervalSec=25s # <1>
 EOF
 ----
+<1> Increase the `StartLimitInterval` value from the default 10s to 25s for slower systems. A value too low can result in systemd never marking `microshift.service` as failed and the `OnFailure=` service not being triggered.
```

Comment:
25s was working for QE envs, but I wouldn't say it's an universal value.

Reply:
I was wondering that.